starter-code/iteration-1/main.ts

```diff
@@ -1,8 +1,51 @@
 // Iteration 1
 // 1. Create an array of strings
-
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
 
+let taskList = [];
```

Comment:
It would be better to wrap everything in a class.
Furthermore, you didn't put the type:
```
let taskList: Array<string> = [];
```